model/project_aliases.go

```diff
@@ -72,33 +72,145 @@ type ProjectAliases []ProjectAlias
 
 // FindAliasesForProject fetches all aliases for a given project
 func FindAliasesForProject(projectID string) ([]ProjectAlias, error) {
```

Comment:
Maybe this should say "FindAllAliasesForProject" or "FindAliasesForProjectWithYamlAliases" ? I feel like both of those names aren't great but to get across that this isn't what it used to be. (Alternatively maybe just a comment would do the trick)

Reply:
I think both would be good; changed name to FindAllAliasesForProject and added comment.